src/applications/disability-benefits/all-claims/tests/all-claims.cypress.spec.js

```diff
@@ -0,0 +1,107 @@
+import path from 'path';
+
+import testForm from 'platform/testing/e2e/cypress/support/form-tester';
+import { createTestConfig } from 'platform/testing/e2e/cypress/support/form-tester/utilities';
+
+import formConfig from '../config/form';
+import manifest from '../manifest.json';
+import { mockItf } from './all-claims.cypress.helpers';
+
+const testConfig = createTestConfig(
+  {
+    dataPrefix: 'data',
+    dataSets: ['maximal-test', 'minimal-test', 'newOnly-test'],
+    fixtures: {
+      data: path.join(__dirname, 'data'),
+      mocks: path.join(__dirname, 'fixtures', 'mocks'),
+    },
+    pageHooks: {
+      introduction: () => {
+        // Hit the start button
+        cy.findAllByText(/start/i, { selector: 'button' })
+          .first()
+          .click();
+
+        // Click past the ITF message
+        cy.findByText(/continue/i, { selector: 'button' }).click();
+      },
+      'disabilities/rated-disabilities': () => {
+        cy.get('@testData').then(data => {
+          data.ratedDisabilities.forEach((disability, index) => {
+            if (disability['view:selected']) {
+              cy.get(`input[name="root_ratedDisabilities_${index}"]`).click();
+            }
+          });
+          cy.findByText(/continue/i, { selector: 'button' }).click();
+        });
+      },
+      'payment-information': () => {
+        cy.get('@testData').then(data => {
+          if (data['view:bankAccount']) {
+            cy.get('form.rjsf').then($form => {
+              const editButton = $form.find('.usa-button-primary.edit-button');
+              if (editButton) editButton.click();
+            });
+
+            cy.fillPage();
+            cy.findByText(/save/i, { selector: 'button' }).click();
+          }
+
+          cy.findByText(/continue/i, { selector: 'button' }).click();
+        });
+      },
+    },
+    setupPerTest: () => {
+      cy.login();
```

Comment:
Some forms will have different flows depending if you're signed in or not. Some can't even be started without being signed in.

This custom command simulates a logged in session.